Advertools/Advertools_Audit_robots_txt_and_xml_sitemap_issues.ipynb

```diff
@@ -177,10 +177,14 @@
    "cell_type": "code",
    "execution_count": 1,
    "id": "4949b9c4-f852-47f5-9889-3ccce1a0100a",
+<<<<<<< HEAD
```

Comment:
@FlorentLvr @eliasdabbas the CI/CD is failing because this notebook is broken. Seems like merge conflict were not resolved.